File: app/src/main/java/com/example/builday365/Model/Timeline/Memo.java
package com.example.builday365.Model.Timeline;

import androidx.room.Entity;
import androidx.room.PrimaryKey;

import java.util.Date;

public class Memo {
    private Date timeStamp;
    private String memoContent;

    public Memo(Date timeStamp, String memoContent) {
        this.timeStamp = timeStamp;
        this.memoContent = memoContent;
    }

    public Date getTimeStamp() {
        return timeStamp;
    }

    public void setTimeStamp(Date timeStamp) {
        this.timeStamp = timeStamp;
    }

    public String getMemoContent() {
        return memoContent;
    }

    public void setMemoContent(String memoContent) {
        this.memoContent = memoContent;
    }

    @Override
    public String toString() {
        return "Memo{" +
                "timeStamp=" + timeStamp +
                ", memoContent='" + memoContent + '\'' +
                '}';
    }
}

File: app/src/main/java/com/example/builday365/model/test/Memo.kt
package com.example.builday365.model.test

import java.util.Date

class Memo(var startTime: Date, var endTime: Date, var memoContent: String) {

    override fun toString(): String {
        return "Memo{" +
                "startTime=" + startTime +
                ", endTime=" + endTime +
                ", memoContent='" + memoContent + '\'' +
                '}'
    }
}